crates/bevy_feathers/src/font_styles.rs

```diff
@@ -17,7 +16,7 @@ use crate::{handle_or_path::HandleOrPath, theme::ThemedText};
 /// downward to any child text entity that has the [`ThemedText`] marker.
 #[derive(Component, Default, Clone, Debug, Reflect)]
 #[reflect(Component, Default)]
-#[require(ThemedText, PropagateOver::<TextFont>::default())]
+#[require(ThemedText)]
```

Comment:
Do we even need `ThemedText` any more?

Reply:
Doesn't look like it, I thought I'd leave it for now though, and focus this PR on the text changes.